src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -303,6 +308,158 @@ public override bool IsSealed
+        {
+            get
+            {
+                return _underlyingType.TypeArgumentsCustomModifiers;
+            }
+        }
+
+        internal override bool HasTypeArgumentsCustomModifiers
+        {
+            get
+            {
+                return _underlyingType.HasTypeArgumentsCustomModifiers;
+            }
+        }
+
+        internal override ImmutableArray<TypeSymbol> TypeArgumentsNoUseSiteDiagnostics
+        {
+            get
+            {
+                return _underlyingType.TypeArgumentsNoUseSiteDiagnostics;
+            }
+        }
+
+        public override NamedTypeSymbol ConstructedFrom
+        {
+            get
+            {
+                return _underlyingType.ConstructedFrom;
+            }
+        }
+
+        public override bool MightContainExtensionMethods
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override string Name
+        {
+            get
+            {
+                return string.Empty;
+            }
+        }
+
+        internal override bool MangleName
+        {
+            get
+            {
+                return _underlyingType.MangleName;
+            }
+        }
+
+        public override IEnumerable<string> MemberNames
+        {
+            get
+            {
+                return _fields.Select(f => f.Name);
+            }
+        }
+
+        internal override bool HasSpecialName
+        {
+            get
+            {
+                return _underlyingType.HasSpecialName;
+            }
+        }
+
+        internal override bool IsComImport
+        {
+            get
+            {
+                return _underlyingType.IsComImport;
+            }
+        }
+
+        internal override bool IsWindowsRuntimeImport
+        {
+            get
+            {
+                return _underlyingType.IsWindowsRuntimeImport;
+            }
+        }
+
+        internal override bool ShouldAddWinRTMembers
+        {
+            get
+            {
+                return _underlyingType.ShouldAddWinRTMembers;
+            }
+        }
+
+        internal override bool IsSerializable
+        {
+            get
+            {
+                return _underlyingType.IsSerializable;
+            }
+        }
+
+        internal override TypeLayout Layout
+        {
+            get
+            {
+                return _underlyingType.Layout;
+            }
+        }
+
+        internal override CharSet MarshallingCharSet
+        {
+            get
+            {
+                return _underlyingType.MarshallingCharSet;
+            }
+        }
+
+        internal override bool HasDeclarativeSecurity
+        {
+            get
+            {
+                return _underlyingType.HasDeclarativeSecurity;
+            }
+        }
+
+        internal override bool IsInterface
+        {
+            get
+            {
+                return _underlyingType.IsInterface;
```

Comment:
I am not sure it is appropriate to delegate to the _underlyingType for the 4 properties above.